eng/tools/mcp/azure-sdk-python-mcp/main.py

```diff
@@ -249,7 +249,33 @@ def init_local_tool(tsp_config_path: str, repo_path:str) -> Dict[str, Any]:
             "stderr": "",
             "code": 1
         }
+    
+@mcp.tool("update")
+def update_tool(commit_hash: str, package_path:str) -> Dict[str, Any]:
+    """Initializes and subsequently generates a typespec client library directory from a local azure-rest-api-specs repo.
+
+    This command is used to update a client library to a specific commit hash in the azure-rest-api-specs repository.
```

Comment:
This is kind of contradicting the previous description. Are we updating based on a local spec or a specific commit hash in the repo? It's fine if this tool can do both but we should clarify what the purpose of this tool is